pkg/action/cl001/ac001/executor.go

```diff
@@ -28,10 +28,9 @@ func (e *Executor) execute(ctx context.Context) error {
 	var crs v1alpha2.ClusterCRs
 	{
 		c := v1alpha2.ClusterCRsConfig{
-			ClusterID:         "cl001",
 			Domain:            key.DomainFromHost(e.clients.ControlPlane.RESTConfig().Host),
-			Description:       "awscnfm cluster cl001",
-			Owner:             "giantswarm",
+			Description:       explainerCommand,
+			Owner:             key.Organization,
```

Comment:
This enables us to more easily add and maintain the cluster creation code. The `explainerCommand` was a quick idea on my side and we will see if this is really useful. I thought with that you can look at a cluster description and execute the explainer command in order to understand what this conformance test cluster is about.